src/discounts/discounts.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { CreateDiscountDto } from './dto/create-discount.dto';
+import { UpdateDiscountDto } from './dto/update-discount.dto';
+
+@Injectable()
+export class DiscountsService {
+  create(createDiscountDto: CreateDiscountDto) {
+    return 'This action adds a new discount';
+  }
+
+  findAll() {
+    return `This action returns all discounts`;
+  }
+
+  findOne(id: number) {
+    return `This action returns a #${id} discount`;
+  }
+
+  update(id: number, updateDiscountDto: UpdateDiscountDto) {
+    return `This action updates a #${id} discount`;
+  }
+
+  remove(id: number) {
+    return `This action removes a #${id} discount`;
+  }
```

Comment:
for not-done feature, it's better to throw error instead of return value to get developers awareness